park-dashboard/src/Pages/NationalParkCharts/NationalParkChartsPage.js

```diff
@@ -3,27 +3,36 @@ import { Container, Row, Col } from "react-bootstrap";
 import ActivitiesVsNP from "../../Components/NationalParkActivities/ActivitiesVsNP";
 import StateVsNationalParks from "../../Components/BarChartStateVsNationalPark/StateVsNationalParks";
 import "./NationalParkChart.css";
+import AlertsVsNationalParks from "../../Components/Alerts3dPieChart/AlertsVsNationalParks";
 const NationalParkChartPage = () => {
   return (
     <div style={{ marginTop: "6rem" }}>
       <h1 className="text-center" style={{ marginBottom: "5rem" }}>
         Visualizations of all parks
       </h1>
-      <Container className="barchart" centered style={{ marginBottom: "4rem"}}>
+      <Container className="barchart" centered style={{ marginBottom: "4rem" }}>
         <Row>
           <h2 style={{ fontSize: "18px" }} className="text-center">
             Bar Chart: State Vs Park Counts
           </h2>
           <StateVsNationalParks />
         </Row>
       </Container>
-      <Container className="donut-chart" centered>
+      <Container>
         <Row>
-          <h2 style={{ fontSize: "18px" }} className="text-center" >
-            Donut Chart: Activities vs National Parks
-          </h2>
-          <ActivitiesVsNP className="py-5 activities" />
+          <Col>
+            <h2 style={{ fontSize: "18px" }} className="text-center">
+              Donut Chart: Activities vs National Parks
+            </h2>
+            <ActivitiesVsNP className="py-5 activities" />
+          </Col>
         </Row>
+        <Col>
+          <h2 style={{ fontSize: "18px" }} className="text-center">
+            3D Pie Chart: Alerts Vs National Park Counts
+          </h2>
+          <AlertsVsNationalParks className="alerts" />
```

Comment:
I remember adding some code to make the charts more responsive? Can you pull the latest main branch and then modify your code here? 

This is my PR where I added code to make the charts responsive -> https://github.com/rutujapadgilwar/Park-Dashboard/pull/13/files#diff-110bc809b1b51a37356fc9ef0586e5e43f81888620da72233e2232c0a8e90355

You can probably replace the second col with your component.

<img width="1509" alt="image" src="https://github.com/rutujapadgilwar/Park-Dashboard/assets/114516950/ab99c2b1-7579-4f81-9dae-5d90df64b968">